### ISSUE TITLE
Add Debian 12 `bookworm` release

### ISSUE THREAD

**Issue by rwlodarczyk-xealth** [openresty/docker-openresty#232]
Since Debian 12 `bookworm` was released on 2023-06-10, please release an updated version for this new Debian release.

**Comment by neomantra** [comment]
As one of the unwritten policies of this image is to track Debian `stable`, I'll add it to the build-from-source images... assuming there are no major issues with building it there.

**Comment by neomantra** [comment]
I forgot that we only follow build-from-upstream for Debian.   The build-from-source is for Ubuntu.  I just added some notes to the README to clarify this. 

You can build your own image by cloning this and running:
```
docker build --build-arg RESTY_IMAGE_BASE=debian --build-arg RESTY_IMAGE_TAG=bookworm -f jammy/Dockerfile .
```

I haven't tested that.  Given it is pretty straight-forward (versus managing another set of Dockerfiles), I might consider adding it to the CI/CD.  I need to think of a naming convention to differentiate it.

**Comment by rwlodarczyk-xealth** [comment]
Related: https://github.com/openresty/openresty/issues/901

**Comment by golgote** [comment]
Are you going to add Debian 12 Bookworm here: https://openresty.org/en/linux-packages.html#debian ? I mostly need to be able to easily install the dependencies like:

```
apt-get -y install openresty-zlib openresty-zlib-dev openresty-pcre openresty-pcre-dev
```

Thanks

**Comment by neomantra** [comment]
Hmmm... I was wanting to keep Debian as build-from-upstream, but they still haven't added it.  I'm not involved with that. 

I could make another image flavor, `bullseye-src`... downside is deviation from policy and more flavors to support.   

I have the build command above, for self-service, which uses the `jammy` Dockerfile.

**Comment by neomantra** [comment]
Upstream `bookworm` flavor released in `1.21.4.3-1`.

**Comment by rwlodarczyk-xealth** [comment]
See #242
